spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+describe "the ingredients index page" do
+  it "lists all the ingredients with their attributes" do
+    ground_beef = Ingredient.create(name: "Ground Beef", cost: 4)
+    garlic = Ingredient.create(name: "Garlic", cost: 2)
+    tomato_paste = Ingredient.create(name: "Tomato Paste", cost: 1)
+    
+    visit "/ingredients"
+
+    expect(page).to have_content("Ground Beef")
+    expect(page).to have_content("Garlic")
+    expect(page).to have_content("Tomato Paste")
+    expect(page).to have_content("4")
+    expect(page).to have_content("2")
+    expect(page).to have_content("1")
+  end
```

Comment:
You should aim to make your tests more robust by reducing the amount of ambiguity in your tests. For example, lines 14, 15, and 16 just mention that the numbers 4, 2, and 1 need to show up on the page. These tests are too ambiguous, they could show up as a date, phone number, or address and this would mark this as passing even if it's not what you intended it to pass on.

A more robust test would be similar to something like this:

```ruby
expect(page).to have_content("Cost: 4") # There is no ambiguity in this test since you know exactly where on the page this would be located.
```

Another thing you could do is to use `within` blocks to target a specific spot on the page. I noticed that you used div's with ID's in the view but didn't make use of them with `within` blocks in your tests.